lib/rules/jsx-wrap-multilines.js

```diff
@@ -32,6 +32,7 @@ const PARENS_NEW_LINES = 'Parentheses around JSX should be on separate lines';
 
 module.exports = {
   meta: {
+    type: 'suggestion',
```

Comment:
```suggestion
    type: 'layout',
```
This rule is similar to [arrow-parens](https://github.com/eslint/eslint/blob/master/lib/rules/arrow-parens.js ) and [no-extra-parens](https://github.com/eslint/eslint/blob/master/lib/rules/no-extra-parens.js). Both of them are considered 'layout'.